indico/htdocs/js/indico/widgets/protection_widget.js

```diff
@@ -29,16 +34,16 @@
         }, options);
 
         var inputs = $('input[name=' + options.fieldName + '][id^=' + options.fieldId + ']');
+        var $enableAclLink = $('.enable-acl-link');
```

Comment:
Wow, didn't expect my comment to raise such a discussion. Let's just find a convention and stick with it! Facebook poll: :+1: for `ACL`, :-1: for `Acl` ! Haha